apis/components/v1/dashboard_types.go

```diff
@@ -43,6 +45,9 @@ type DashboardSpec struct {
 // DashboardStatus defines the observed state of Dashboard
 type DashboardStatus struct {
 	components.Status `json:",inline"`
+
+	Namespace string `json:"namespace,omitempty"`
+	URL       string `json:"url,omitempty"`
```

Comment:
Is this the url for manifests or consolelink?

Reply:
The one to reach the dashboard (the same one set in the consolelink). 

I've added it here as an example of how to make things discoverable without having to know the internal details (route, consolelink, etc). 

It can be removed if it does not make sense.